Source/Blazorise/Components/Accordion/AccordionToggle.razor.cs

```diff
@@ -76,6 +88,24 @@ public bool CollapseVisible
         }
     }
 
+    /// <summary>
+    /// Gets or sets the content visibility.
+    /// </summary>
+    [CascadingParameter( Name = "AccordionItemVisible" )]
+    public bool AccordionItemVisible
```

Comment:
Didn't we use a state mechanism for this things before?

Reply:
For Accordion no. We keep it simple until we don't :)